tests/vmi_lifecycle_test.go

```diff
@@ -70,7 +70,7 @@ import (
 	"kubevirt.io/kubevirt/tests/watcher"
 )
 
-func addNodeAffinityToVMI(vmi *v1.VirtualMachineInstance, nodeName string) {
+func AddNodeAffinityToVMI(vmi *v1.VirtualMachineInstance, nodeName string) {
```

Comment:
Maybe move to `libvmi` and rename to `AddNodeAffinity`

Reply:
I was going to do it in a later PR, but I'll add a commit at this point. I am going to drop the entire function, switch using `WithNodeAffinity` Option in `libvmi`